diff --git a/node_modules/lodash.get/index.js b/node_modules/lodash.get/index.js
index 0eaadec..6fcd4d8 100644
--- a/node_modules/lodash.get/index.js
+++ b/node_modules/lodash.get/index.js
@@ -594,7 +594,7 @@ function isMasked(func) {
  * @param {string} string The string to convert.
  * @returns {Array} Returns the property path array.
  */
-var stringToPath = memoize(function(string) {
+var stringToPath = /*memoize(*/function(string) {
   string = toString(string);
 
   var result = [];
@@ -605,7 +605,7 @@ var stringToPath = memoize(function(string) {
     result.push(quote ? string.replace(reEscapeChar, '$1') : (number || match));
   });
   return result;
-});
+}/*)*/;
 
 /**
  * Converts `value` to a string key if it's not a string or symbol.
